oauth2.go

```diff
@@ -16,6 +16,10 @@ func (z *Zoho) SetRefreshToken(refreshToken string) {
 	z.oauth.token.RefreshToken = refreshToken
 }
 
+func (z *Zoho) GetRefreshToken() string {
+	return z.oauth.token.RefreshToken
+}
+
```

Comment:
Please add a comment for godoc. I may have forgotten to ask you in the previous PR